riptide-httpclient/src/main/java/org/zalando/riptide/httpclient/Closing.java

```diff
@@ -0,0 +1,28 @@
+package org.zalando.riptide.httpclient;
+
+import com.google.common.annotations.VisibleForTesting;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.Nullable;
+import java.io.Closeable;
+import java.io.IOException;
+
+@Slf4j
+final class Closing {
+
+    Closing() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/whiskeysierra/riptide/pullRequest?prid=4955663)